### PR TITLE
Bump node version

### PR DESCRIPTION
**Pull request by nokki34**
### Ticket Link

https://app.shortcut.com/amazd/story/6245/move-all-apps-to-latest-node

### Changes

This PR updates the node version

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,5244 +1,8 @@
 {
   "name": "graphql-upload-minimal",
   "version": "1.2.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (deviousdodo) [inline package-lock.json]: seems like we are going back to older versions?

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-v16.13.2
+10
```
Comment (deviousdodo) [inline .nvmrc:1]: node 10?

Comment (akhegai) [reply .nvmrc:1]: yup, made some mess. I'll ping you

**Comment by deviousdodo** [comment]
@akhegai as you upload to latest node, maybe this package is not even required anymore. we had a bug when uploading files, but maybe it's simply solved with latest nodes.

**Pull request closed without merging**